invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -523,7 +611,9 @@ def draw_line(self, pos0, pos1, arrow_start=False, arrow_end=False, colour=(255,
 
         self._drawn = True
 
-    def draw_circle(self, center, radius=2.5, width=2, line_colour=(255, 0, 0, 128), fill_colour=(0, 0, 0, 0)):
+    def draw_circle(
+        self, center, radius=2.5, width=2, line_colour=(255, 0, 0, 128), fill_colour=(0, 0, 0, 0)
```

Comment:
center is tuple of two integers
radius is float
width is integer
line_colour is tuple of 4 integers
fill_colour is tuple of 4 integers

Reply:
I think center should  be a `Tuple[float, float]`

Reply:
Yeah, you are correct. Also, the [circle creation](https://docs.wxpython.org/wx.GraphicsPath.html#wx.GraphicsPath.AddArc) accepts float. Every position parameter here in this class and file can be float.